worlds/mm2/docs/en_Mega Man 2.md

```diff
@@ -0,0 +1,114 @@
+# Mega Man 2
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Weapons received from Robot Masters, access to each individual stage, and Items from Dr. Light are randomized
+into the multiworld. Access to the Wily Stages is locked behind receiving Item 1, 2, and 3. The game is complete upon 
+viewing the ending sequence after defeating the Alien.
+
+## What Mega Man 2 items can appear in other players' worlds?
+- Robot Master weapons
+- Robot Master Access Codes (stage access)
+- Items 1/2/3
+- 1-Ups
+- E-Tanks
+- Health Energy (L)
+- Weapon Energy (L)
+
+## What is considered a location check in Mega Man 2?
+- The defeat of a Robot Master or Wily Boss
+- Receiving a weapon or item from Dr. Light
+- Optionally, 1-Ups and E-Tanks present within stages
+- Optionally, Weapon and Health Energy pickups present within stages
+
+## When the player receives an item, what happens?
+A sound effect will play based on the type of item received, and the effects of the item will be immediately applied, 
+such as unlocking the use of a weapon mid-stage. If the effects of the item cannot be fully applied (such as receiving 
+Health Energy while at full health), the remaining are withheld until they can be applied.
```

Comment:
Not exactly, if you can only apply half of a weapon energy, it'll save the other half until it can be applied. Definitely better wording possible though.